packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -302,12 +303,12 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
             )
         );
 
-        return
-            SecureMerkleTrie.verifyInclusionProof(
-                abi.encode(storageKey),
-                hex"01",
-                _proof,
-                _storageRoot
-            );
+        (bool exists, bytes memory value) = MerkleTrie.get(
```

Comment:
I feel like the previous API is less bug prone

Reply:
An example here is the first argument should be a bytes32 to enforce that a hashed value is passed in. There is no way for the caller to know that a hashed value should be passed in. I like the older API more because its more clear how to use it